package.json

```diff
@@ -55,6 +55,5 @@
     "prettier": "^2.5.1",
     "ts-jest": "^27.1.2",
     "typescript": "4.5.4"
-  },
-  "dependencies": {}
```

Comment:
Please also remove this change. (All changes to package.json files, for that matter.)